File: RoboschienWeb/ClientApp/src/app/models/associate.ts
// {
// 	"AssociateDetails": {
// 			"AssociateNumber" : "50697851",
// 			"AssociateFirstName": "Suresh",
// 			"AssociateLastName": "Kona",
// 			"AssociateEmail": "suresh.kona@in.bosch.com",
// 			"AssociateMobileNumber": "+49 89-636-48018"
// 		}
// 	"WorkDisabilityCertificateDetails" : {
// 			"FirstTimeSickness" : true,
// 			"FollowUpSickness" : false,
// 			"Accident" : false,
// 			"StartDate" : "2018-07-07",
// 			"EndDate" : "2018-07-20",
// 		}
// }
import{WorkDisabilityCertificate} from './WorkDisabilityCertificate';
export class AssociateDetails{
    
   AssociateNumber:string;
   AssociateFirstName:string;
   AssociateLastName:string;
   AssociateEmail:string;
  AssociateMobileNumber: string;
  DataPrivacyTimeStamp: string;
  SelectedCountryCode: string;
  IsOcrCheck: boolean;

   WorkDisabilityCertificateDetails:WorkDisabilityCertificate;
  
}

File: RoboschienWeb/Models/Entities.UI/Associate.cs
using Newtonsoft.Json;

namespace RoboschienWeb.Models.Entities.UI
{
    public class Associate
    {
        public string AssociateNumber { get; set; }
        public string AssociateFirstName { get; set; }
        public string AssociateLastName  { get; set; }
        public string AssociateEmail {get; set; }
        public string AssociateMobileNumber { get; set; }
        public string DataPrivacyTimeStamp { get; set; }

        public string SelectedCountryCode { get; set; }
        public bool IsOcrCheck { get; set; }



        [JsonProperty("WorkDisabilityCertificateDetails")]
        public WorkDisabilityCertificateDetails workDisabilityCertificateDetailsInfo { get; set; }
       
    }
}